WordPress/pattern-directory
# Patterns list table improvements

* Add columns to the patterns list table for each of the related taxonomies.
* Add a column to the patterns list table that shows how many pending flags a pattern has.
* Enable filtering by status when viewing flags for only one pattern.

Refs #63

This doesn't cover the requirement listed in #63 to add a modal for giving a reason when changing a pattern's status.


### Screenshots

![pattern-list-table](https://user-images.githubusercontent.com/916023/117738566-30bdb980-b1b1-11eb-910b-934fa4d1289d.jpg)

---

![flags-list-table](https://user-images.githubusercontent.com/916023/117738579-361b0400-b1b1-11eb-9957-1a813ae28a41.jpg)


### How to test the changes in this Pull Request:

1. Add some flags to some patterns. See [these testing instructions](https://github.com/WordPress/pattern-directory/pull/77) if #83 isn't merged yet.
2. View the patterns list table.
3. Try clicking the Has Flags view to see only patterns that have "pending" flags.
3. Try clicking on one of the numbers in the Pending Flags column to view flags for just that pattern.
4. Change the status of a few flags so you can see the view links at the top of the patterns list table change.
